RockWeb/Blocks/Finance/BusinessList.ascx

```diff
@@ -30,8 +30,8 @@
                             <Rock:RockTemplateField>
                                 <HeaderTemplate>Contact Information</HeaderTemplate>
                                 <ItemTemplate>
-                                    <%# Eval("PhoneNumber") %><br />
-                                    <%# Eval("Email") %>
+                                    <%# Eval("PhoneNumber") %>  <%# !string.IsNullOrEmpty( (string)Eval( "PhoneNumber" ) )? "<br />":string.Empty %>
+                                    <%# Eval("Email") %> <%# string.IsNullOrEmpty( (string)Eval( "PhoneNumber" ) )? "<br />&nbsp; ":string.Empty %>
```

Comment:
@shivambareria Not sure why the email would ever need the `<br/>&nbsp`, and it's also checking  if PhoneNumber is empty.  Probably just need the change on line #33 (and not the change on #34).